lib/presentation/bill_details/components/plan_table.dart

```diff
@@ -0,0 +1,182 @@
+                                  .days,
+                              style: planTextStyle,
+                            ),
+                            Column(
+                              mainAxisAlignment: MainAxisAlignment.center,
+                              crossAxisAlignment: CrossAxisAlignment.start,
+                              children: [
+                                const Gap(8),
+                                Text(
+                                  t.billDetailsPage.pricingPlan.details
+                                      .premiumPrice.daily,
+                                  style: planTextStyle,
+                                ),
+                                Text(
+                                  t.billDetailsPage.pricingPlan.details
+                                      .premiumPrice.threeDays,
+                                  style: planTextStyle,
+                                ),
+                                Text(
+                                  t.billDetailsPage.pricingPlan.details
+                                      .premiumPrice.fiveDays,
+                                  style: planTextStyle,
+                                ),
+                                Text(
+                                  t.billDetailsPage.pricingPlan.details
+                                      .premiumPrice.sevenDays,
+                                  style: planTextStyle,
+                                ),
+                              ],
+                            ),
+                          ],
+                        ),
+                        const Gap(12),
+                        Text(
+                          t.billDetailsPage.pricingPlan.details.premiumPrice.or,
+                          style: AppTextStyle.textStyle.copyWith(
+                            fontSize: 12,
+                          ),
+                        ),
+                        const Gap(12),
+                        Column(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            Text(
+                              t.billDetailsPage.pricingPlan.details.premiumPrice
+                                  .lifetime,
+                              style: planTextStyle,
+                            ),
+                            const Gap(12),
+                            Column(
+                              mainAxisAlignment: MainAxisAlignment.center,
+                              crossAxisAlignment: CrossAxisAlignment.start,
+                              children: [
+                                Text(
+                                  t.billDetailsPage.pricingPlan.details
+                                      .premiumPrice.lifetimePrice,
+                                  style: planTextStyle,
+                                ),
+                              ],
+                            ),
```

Comment:
Columnウィジェットごといらないかも？
```suggestion
                                Text(
                                  t.billDetailsPage.pricingPlan.details
                                      .premiumPrice.lifetimePrice,
                                  style: planTextStyle,
                                ),
```

Reply:
一応ローカルで消して試したけれど、なくても大丈夫そう

Reply:
こちらも確認して削除しても変わらなかったので削除しました！ありがとう！Done!